tests/snapshots/generate_plan_tests__node_canvas.snap

```diff
@@ -17,10 +17,10 @@ expression: plan
         "setup"
       ],
       "commands": [
-        "npm i"
+        "pnpm i --frozen-lockfile"
```

Comment:
The `node-canvas` example has all types of lockfiles. This isn't that realistic and it should probably just have a single one.

Reply:
Yea fair nuff